pkg/operator/controller/credentials-secret/controller.go

```diff
@@ -200,7 +261,10 @@ func isInNS(namespace string) func(o client.Object) bool {
 }
 
 // getExternalDNSCredentialsSecretName returns the name of the credentials secret retrieved from externalDNS resource
-func getExternalDNSCredentialsSecretName(externalDNS *operatorv1alpha1.ExternalDNS) string {
+func (r *reconciler) getExternalDNSCredentialsSecretName(externalDNS *operatorv1alpha1.ExternalDNS) string {
```

Comment:
should this function still be a method of the reconciler? I think it's not needed anymore as `IsOpenShift` is global now, no?